packages/ppx_factory/ppx_factory.0.2.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "Cryptosense <opensource@cryptosense.com>"
+authors: [
+    "Cryptosense <opensource@cryptosense.com>"
+    "Nathan Rebours <nathan.p.rebours@gmail.com>"
+]
+homepage: "https://github.com/cryptosense/ppx_factory"
+bug-reports: "https://github.com/cryptosense/ppx_factory/issues"
+license: "BSD-2"
+dev-repo: "git+https://github.com/cryptosense/ppx_factory.git"
+doc: "https://cryptosense.github.io/ppx_factory/doc"
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+run-test: [
+  [ "dune" "runtest" "-p" name "-j" jobs ]
+]
+depends: [
+  "dune" {>= "1.1"}
+  "ocaml" {>= "4.07.0" & < "4.11.0"}
```

Comment:
Is this correct? It does not support 4.11?

Reply:
I haven't tried locally and Travis CI wouldn't tell me. (https://travis-ci.com/github/cryptosense/ppx_factory/builds/199372106) Until I fixed that I figured that I would submit the package as it is.

That being said, I think I was told I could remove the upper bound optimistically for future package submissions.

Reply:
We can test it with the local CI and then re-add the upper bound in case it fails if you want

Reply:
```suggestion
  "ocaml" {>= "4.07.0" & < "4.12.0"}
```

Reply:
Sounds good to me!